e2e/jest.test-setup.ts

```diff
@@ -14,9 +12,9 @@ const isDebugMode = process.argv.includes('--debug');
  */
 beforeEach(async () => {
   await page.setUserAgent(userAgent);
-  // See https://github.com/puppeteer/puppeteer/blob/master/docs/api.md#pagesetdefaultnavigationtimeouttimeout
-  page.setDefaultNavigationTimeout(navTimeOut);
-  page.setDefaultTimeout(timeOut);
+  await page.setViewport({width: 1280, height: 680});
+  page.setDefaultNavigationTimeout(60000); // Puppeteer default timeout is 30 seconds.
```

Comment:
Is there a way to collect data on how long some of these actions take? Like record them to a spreadsheet or something? It would be interesting to see if 80% were under 2 seconds or something.

Reply:
there is a way by Puppeteer. I am taking note on this and a future PR is brewing.